.github/workflows/deploy.yml

```diff
@@ -37,24 +37,14 @@ jobs:
                 SE-SECRETS 
                 SE-INFRA-SECRETS 
 
-  turnstyle:
-    name: Wait for any previous deployments
-    timeout-minutes: 20
-    runs-on: ubuntu-latest
-    needs: validate
-    steps:
-      - name: Turnstyle
-        uses: softprops/turnstyle@v1
-        with:
-          continue-after-seconds: 300
-          same-branch-only: false
-
   deploy:
     name: Deploy ${{ github.event.inputs.sha }} to ${{ github.event.inputs.environment }}
     environment: 
        name: ${{ github.event.inputs.environment }}
     runs-on: ubuntu-latest
-    needs: turnstyle
+    needs: validate
+
+    concurrency: ${{github.event.inputs.environment}}
```

Comment:
Since 2 distinct PRs have the same "Review" environment, I understand it will prevent parallel 2 deployments. Could we add `-${{github.event.inputs.pr}}`? It should solve this issue for PRs and hopefully it's null or empty for regular environments.

Reply:
The calling function uses 
```
      - name: Trigger Deployment to ${{matrix.environment}}
        uses: benc-uk/workflow-dispatch@v1.1
        with:
          workflow: Deploy to PaaS
          token: ${{ steps.ACTIONS-API-ACCESS-TOKEN.outputs.secret-value }}
          inputs: '{"environment": "${{matrix.environment}}", "sha": "${{ github.sha }}" , "pr": "${{github.event.number}}" , "static": "${{env.STATIC_ROUTE}}" }'
          ref: ${{env.REFERENCE}}
```
which i could change but would need an IF statement and this called different ways for Review and Development/Test/Production. Which I would prefer not to. Also I am concerned the destroy does not know the PR number